### ISSUE TITLE
error by running audio_duration

### ISSUE THREAD

**Issue by toutPetitPoissonRouge** [p32929/flutter_audio_duration#1]
hi,
thanks for sharing this package... i installed it and run it. It compiles but throw following error in the console when the app is running:

'An error occurred: MissingPluginException(No implementation found for method getAudioDuration on channel audio_duration)'

i imported the package in my main.dart and installed the 0.0.3 (audio_duration: ^0.0.3) verion in pubspec.yaml.

the line code calling the fonction is: 

'var silenceDurationInMilliSeconds = await AudioDuration.getAudioDuration('assets/s100ms.mp3'); 
 print('silence Duration = $silenceDurationInMilliSeconds');'

  any idea why ?

thanks again,

**Comment by p32929** [comment]
may I ask, which platform have you been trying it on?

**Comment by toutPetitPoissonRouge** [comment]
yes i am develepping in visual studio code on my PC and i test the app in Chrome. 
My goal is to developp a flutter-web-app.

**Comment by p32929** [comment]
oww. currently it only supports Android

![image](https://github.com/p32929/flutter_audio_duration/assets/6418354/7e0d6c12-b11b-4f4f-a488-7e2872877edd)

**Comment by p32929** [comment]
this package only works on Android devices. it wont work on Web or any other platforms.

![image](https://github.com/p32929/flutter_audio_duration/assets/6418354/a816725a-8d02-49e8-987a-f384964e4d46)

**Comment by toutPetitPoissonRouge** [comment]
ok fine. thanks for the reply. Do you foresee a further developpement for other platforms ?

**Comment by toutPetitPoissonRouge** [comment]
i would be intereseted in...

**Comment by p32929** [comment]
yes, I'm actually planning to support it for the web, not sure about iOS though as I dont have an iOS/mac device

**Comment by p32929** [comment]
yes, I have plans to do it. but I'm too busy with my schedules. But forks & pull requests are welcome
